ansible/roles/h5p-library-management/templates/api-h5p-library-management-cronjob.yml.j2

```diff
@@ -3,15 +3,15 @@ kind: CronJob
 metadata:
   namespace: {{ NAMESPACE }}
   labels:
-    app: api-library-management-cronjob
+    app: api-h5p-library-management-cronjob
     app.kubernetes.io/part-of: schulcloud-verbund
     app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
-    app.kubernetes.io/name:  api-library-management-cronjob
+    app.kubernetes.io/name:  api-h5p-library-management-cronjob
     app.kubernetes.io/component: h5p
     app.kubernetes.io/managed-by: ansible
     git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
     git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
-  name: api-library-management-cronjob
+  name: api-h5p-library-management-cronjob
```

Comment:
don't rename stuff esealy the old stuff is still there if it is one time deployed